apps/judicial-system/backend/src/app/modules/subpoena/test/internalSubpoenaController/deliverSubpoenaToPolice.spec.ts

```
@@ -0,0 +1,77 @@
+
+  let mockSubpoenaService: SubpoenaService
+  let givenWhenThen: GivenWhenThen
+
+  beforeEach(async () => {
+    const { subpoenaService, internalSubpoenaController } =
+      await createTestingSubpoenaModule()
+
+    mockSubpoenaService = subpoenaService
+
+    const deliverSubpoenaToPoliceMock =
+      (mockSubpoenaService.deliverSubpoenaToPolice = jest.fn())
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)